src/chromeServices/DOMEvaluator.ts

```diff
@@ -22,13 +22,20 @@ const handleMessages = (
 
       // TODO: Improve error handling
       // - Display failed downloads
-      getCredentials()
+      getCredentials(msg.message)
```

Comment:
`msg.message` does not feel natural, maybe we should adjust the DOMMessage type?